azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestUtil.java

```diff
@@ -16,18 +16,20 @@
 
 package azkaban.execapp;
 
+import azkaban.executor.ExecutableFlow;
 import azkaban.flow.Flow;
 import azkaban.project.DirectoryFlowLoader;
 import azkaban.project.Project;
 import azkaban.project.ProjectManagerException;
+import azkaban.utils.JSONUtils;
 import azkaban.utils.Props;
 import java.io.File;
 import java.io.IOException;
 import java.util.Map;
 import org.apache.commons.io.FileUtils;
 import org.apache.log4j.Logger;
 
-class FlowRunnerTestUtil {
+public class FlowRunnerTestUtil {
```

Comment:
Why isn't package private enough?

Reply:
Because `RemoteFlowWatcherTest` is in `azkaban.execapp.event`. I tried, it can't see this class if it's package visible.